src/plugins/maps_legacy/public/leaflet.js

```diff
@@ -31,6 +29,6 @@ if (!window.hasOwnProperty('L')) {
   require('leaflet-draw/dist/leaflet.draw.js');
   require('leaflet-responsive-popup/leaflet.responsive.popup.css');
   require('leaflet-responsive-popup/leaflet.responsive.popup.js');
-} else {
-  L = window.L;
 }
+
+export const L = window.L;
```

Comment:
Fun little side effect of the bundle sharing, this code used to rely on a bundle that has a dependency on `mapsLegacy` loading leaflet onto `window.L` but not importing it, as the mutable export only worked in bundles which followed the second branch of the if statement. Then the bundles which do import it would see `window.L` is already defined and define the mutable export 🤹